### DISCUSSION TITLE
Service account requiring local administrator

### DISCUSSION THREAD

**Discussion by tfbryan** [datalust/seq-tickets#1397] [Help] [answered]
It seems that we've always had our seq instance running under a service account that was granted local administrator privileges. We're trying to move to a new Group Managed Service Account but cannot get the seq service to remain started unless we again grant local administrator access. The new account has "Logon as a Service" granted, and I've tried adding http reservations but it didn't seem to help. Are there any other suggestions or requirements? Just in case it makes any difference, we've recently configured seq HA. 

I saw this in some of the online documentation. Is this recommended for this situation, and if so, will it require me to perform all the HA configuration again?

_"If Seq is running as a restricted account, the quickest way to ensure that all appropriate permissions are applied is to re-install Seq with the new details:"_

Thanks so much, 

Todd

**Comment by nblumhardt** [comment, marked as answer]
Hi Todd,

Thanks for dropping us a line! The service account will also need to be granted read/write access to the Seq storage folder, e.g. `C:\ProgramData\Seq` (which you can find by running `seq service list`).

If that doesn't help, reinstalling the service will usually set things up successfully. You won't need to reconfigure everything in this case, but do make sure that the `--storage` argument is supplied and points to the same location as above:

```
seq service stop
seq service uninstall
seq service install -u "DOMAIN\user" --storage="C:\ProgramData\Seq"
seq service start
```

Let me know if this helps, or if you need any additional info.

Best regards,
Nick

**Reply by tfbryan** [reply]
Nick,

Thanks so much for the quick response! I was finally able to get the service to remain started after granting the Read/Write access that you suggested. I granted access to `D:\ProgramData\Seq` since that is where this installation is running, but I also granted at `C:\ProgramData\Seq` because that is where the Log folder seems to be.

**Reply by nblumhardt** [reply]
Perfect - thanks for the follow-up, Todd!


Best regards,
Nick
